src/Command/PluginUnloadCommand.php

```diff
@@ -70,7 +70,8 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
      */
     protected function modifyApplication(string $app, string $plugin): bool
     {
-        $finder = "@\\\$this\-\>addPlugin\(\s*'$plugin'(.|.\n|)+\);+@";
+        $finder = "/\s*\\\$this\-\>addPlugin\(\s*['\"]{$plugin}['\"](\s*,[\s\\n]*\[(\\n.*|.*){0,5}\][\\n\s]*)?\);/m";
+        // $finder = "@\\\$this\-\>addPlugin\(\s*'$plugin'(.|.\n|)+\);+@";
```

Comment:
Commented code needs to be removed :)

Reply:
whoops. Was having a hard time with this regex.